plugins/googleanalytics/src/org/pentaho/di/trans/steps/googleanalytics/GaInputStep.java

```diff
@@ -26,6 +26,7 @@
 import java.util.List;
 
 import com.google.api.services.analytics.Analytics;
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
We've used this annotation quite a few other places in kettle, should this be discouraged?